app/apphandlers/gatewayAppHandler.go

```diff
@@ -108,9 +109,10 @@ func (a *gatewayApp) StartRudderCore(ctx context.Context, options *app.Options)
 	})
 
 	var gw gateway.HandleT
-	var rateLimiter ratelimiter.HandleT
-
-	rateLimiter.SetUp()
+	rateLimiter, err := gwThrottler.New(stats.Default)
+	if err != nil {
+		return fmt.Errorf("failed to create rate limiter: %w", err)
+	}
```

Comment:
Should we have a way to fall back to the legacy throttler? 
What is the risk of introducing GCRA new component on the gateway? 
Could this increase the load on redis?

Reply:
I don't think we're introducing the distributed one here, right @cisse21 ?

Reply:
We are not using the distributed one here... We can altogether disable the rate limiter in case we run into any issues. Currently we use this only on hosted and no other customer we have rate limiter enabled